app/components/scbd-angularjs-services/services/storage.js

```diff
@@ -589,7 +602,7 @@
                     "metadata": metadata
                 };
 
-                var useCache = !!params.cache;
+                var useCache = true;//!!params.cache;
```

Comment:
Isn't to systematically for using cache from the Service? I think the user must decide

Reply:
fixed